### DISCUSSION TITLE
Deploying to Vercel and Netlify: This page is taking too long to load.  (Unicorn)

### DISCUSSION THREAD

**Discussion by johncanero** [community/community#35856] [Other Feature Feedback, Questions, & Ideas]
Hello GitHub Community, 

Could someone help me solve this problem. I cannot add a new repository for deployment in Vercel or Netlify and it shows me this unicorn. What seems to be the problem? Need some help! Thank you. 

<img width="596" alt="VERCEL" src="https://user-images.githubusercontent.com/99738754/195240109-e4988dd2-1d5d-4712-9ab5-1065192a232c.png">

**Comment by mr-palindrome** [comment]
I'm facing the same issue with my primary account. While using any other account it's working fine!

**Comment by mr-palindrome** [comment]
I think I found the solution. Try leaving those organisation where you are collaborator!
It worked for me!

**Reply by avishekdas2018** [reply]
I am facing this problem with render. Could you explain your solution bro?
